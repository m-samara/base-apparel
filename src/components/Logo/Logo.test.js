import { render } from '@testing-library/react';
import Logo from './Logo';
 
describe('<Logo />', () => {
 
    it('renders the logo', () => {
        const {container} = render(<Logo/>);
        expect(container).toMatchSnapshot();
    });
});