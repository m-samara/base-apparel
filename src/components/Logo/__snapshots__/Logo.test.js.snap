// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders the logo 1`] = `
<div>
  <div
    class="logo"
  >
    OBASE APPAREL
  </div>
</div>
`;
